nnsvs/gen.py

```diff
@@ -160,17 +188,41 @@ def predict_duration(device, labels, duration_model, duration_in_scaler, duratio
     # Apply model
     x = torch.from_numpy(duration_linguistic_features).float().to(device)
     x = x.view(1, -1, x.size(-1))
-    pred_durations = duration_model(x, [x.shape[1]]).squeeze(0).cpu().data.numpy()
 
-    # Apply denormalization
-    pred_durations = duration_out_scaler.inverse_transform(pred_durations)
+    if duration_model.prediction_type == "probabilistic":
```

Comment:
Same as above.

Reply:
I changed to use  predictin_type() of BaseModel in the latest commit.